src/main/resources/default/view/parts/formBar.html

```diff
@@ -1,11 +1,26 @@
-@args String backUrl
+@args String backUrl, boolean singleClick
 <div class="form-actions">
-    <a onclick="$('#editForm').submit()" class="btn btn-primary">
+    <a href="javascript:$('#editForm').submit()" class="btn btn-primary @if (singleClick) { single-click-link }">
         <i class="fa fa-check"></i> @i18n("NLS.save")
     </a>
     @if (Strings.isFilled(backUrl)) {
         <a href="@prefix/@backUrl" class="btn">@i18n("NLS.back")</a>
     }
 
     @renderBody()
-</div>
\ No newline at end of file
+</div>
+
+@if (singleClick) {
```

Comment:
kann man das net ins template.html schieben? da muss man ja net N mal machen vllt. verwenden wir es ja uach mal in nem anderen button oder so